esphome/components/pvvx_mithermometer/display/__init__.py

```diff
@@ -0,0 +1,60 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import ble_client, display, time
+from esphome.const import (
+    CONF_AUTO_CLEAR_ENABLED,
+    CONF_DISCONNECT_DELAY,
+    CONF_ID,
+    CONF_LAMBDA,
+    CONF_TIME_ID,
+    CONF_VALIDITY_PERIOD,
+)
+
+DEPENDENCIES = ["ble_client"]
+
+pvvx_ns = cg.esphome_ns.namespace("pvvx_mithermometer")
+PVVXDisplay = pvvx_ns.class_(
+    "PVVXDisplay", cg.PollingComponent, ble_client.BLEClientNode
+)
+PVVXDisplayRef = PVVXDisplay.operator("ref")
+
+CONFIG_SCHEMA = (
+    display.BASIC_DISPLAY_SCHEMA.extend(
+        {
+            cv.GenerateID(): cv.declare_id(PVVXDisplay),
+            cv.Optional(CONF_TIME_ID): cv.use_id(time.RealTimeClock),
+            cv.Optional(CONF_AUTO_CLEAR_ENABLED, default=True): cv.boolean,
+            cv.Optional(CONF_DISCONNECT_DELAY, default="5s"): cv.positive_time_period,
+            cv.Optional(CONF_VALIDITY_PERIOD, default="5min"): cv.All(
+                cv.positive_time_period_seconds,
+                cv.Range(max=cv.TimePeriod(seconds=65535)),
+            ),
+        }
+    )
+    .extend(ble_client.BLE_CLIENT_SCHEMA)
+    .extend(cv.polling_component_schema("60s"))
+)
+
+
+async def to_code(config):
+    var = cg.new_Pvariable(config[CONF_ID])
+    await cg.register_component(var, config)
+    await display.register_display(var, config)
+    await ble_client.register_ble_node(var, config)
+    cg.add(var.set_disconnect_delay(config[CONF_DISCONNECT_DELAY].total_milliseconds))
+
+    if CONF_AUTO_CLEAR_ENABLED in config:
+        cg.add(var.set_auto_clear(config[CONF_AUTO_CLEAR_ENABLED]))
```

Comment:
A default is provided so the config options is always there
```suggestion
    cg.add(var.set_auto_clear(config[CONF_AUTO_CLEAR_ENABLED]))
```